src/codemagic/tools/_app_store_connect/arguments.py

```diff
@@ -374,6 +374,7 @@ class BuildArgument(cli.Argument):
         description='Alphanumeric ID value of the Build',
         argparse_kwargs={'required': False},
     )
+    BUILD_ID_RESOURCE_ID_REQUIRED = BUILD_ID_RESOURCE_ID_OPTIONAL.duplicate(argparse_kwargs={'required': True})
```

Comment:
Did you consider using the positional equivalent argument `BuildArgument.BUILD_ID_RESOURCE_ID`?

Reply:
Any thoughts?